frontend/src/scenes/feature-flags/FeatureFlagInstructions.tsx

```diff
@@ -299,44 +190,86 @@ export function CodeInstructions({
                             }}
                             value={selectedOption.value}
                         />
-                        {PAYLOAD_OPTIONS.map((payloadOption) => payloadOption.value).includes(selectedOption.value) && (
+                        <div className="flex items-center gap-1">
+                            <LemonCheckbox
+                                label="Show payload option"
+                                onChange={() => setShowPayloadCode(!showPayloadCode)}
+                                checked={showPayloadCode}
+                                disabled={
+                                    !PAYLOAD_OPTIONS.map((payloadOption) => payloadOption.value).includes(
+                                        selectedOption.value
+                                    )
+                                }
+                            />
+                            <Tooltip
+                                title={
+                                    <>
+                                        {`Feature flag payloads is only available in these libraries: ${PAYLOAD_OPTIONS.map(
+                                            (payloadOption) => payloadOption.value
```

Comment:
needs a space in between
```suggestion
                                            (payloadOption) => `${payloadOption.value}, `
```

Reply:
That doesn't look like what you think it'll look like

<img width="399" alt="Screen Shot 2023-03-14 at 11 52 40 AM" src="https://user-images.githubusercontent.com/25164963/225059676-9f542c0f-e316-4b6a-a6cb-88bb56c05141.png">

There's some code magic that auto adds the commas already to the array of options, so unless it's a big deal I would just leave it?

Reply:
Dang it, wonder how that happens.

Anyways, updated both code suggestions above to something that should work here^^